modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2AnalyticsCloudSetup.macro

```diff
@@ -44,13 +44,13 @@ definition {
 	}
 
 	macro clearAndDeleteCloudEntries {
-		CustomerPortal.clearActivationStatus(
-			position = "${position}",
-			status = "${status}");
-
 		CP2DXPCloudSetup.deleteEntryCloudEnvironments(
 			entry = "${entry}",
 			portlet = "Analytics Cloud Workspaces");
+
+		CustomerPortal.clearActivationStatus(
+			position = "${position}",
+			status = "${status}");
```

Comment:
the macro says clear and delete but the order here is deleting then it clears. Is it correct?

Reply:
oh yes, I hadn't realized. I'll change.